Samples/1.x/ReplicatedEventSample/ReplicatedEventSample.Grains/EventGrain.cs

```diff
@@ -31,7 +31,7 @@ public string EventName
 
         public async Task NewOutcome(Outcome outcome)
         {
-            logger.Info("{3} new outcome {0} {1} {2}", outcome.When, outcome.Name, outcome.Score, EventName);
+            logger.LogInformation("{3} new outcome {0} {1} {2}", outcome.When, outcome.Name, outcome.Score, EventName);
```

Comment:
We need to convert these parameters to names rather than indices. Otherwise, the parameters will not match up. For example `log.LogInformation("{1} {0} {2}", 0, 1, 2)` prints `0 1 2` while `log.Info` for those arguments would have printed `1 0 2`

Reply:
Orleans should be using LoggerMessage.Define or the new source generator.

Reply:
@davidfowl agreed: we should use the new Logging source generator. Is there any more info on it?
LoggerMessage.Define is a lot of boilerplate to put everywhere